### PR TITLE
feat: Show avatar when connected

### PR DESCRIPTION
**Pull request by fzavalia**
image

The change account button just deletes the identity and makes the user sign in again.

Requires https://github.com/decentraland/decentraland-connect/pull/82

### CODE REVIEW

File: src/components/auth/LoginWithAuthServerPage.tsx [hunk 1]
```diff
@@ -1,41 +1,71 @@
...
+          AuthServerProvider.setAuthServerUrl('https://auth-api.decentraland.zone')
+          AuthServerProvider.setAuthDappUrl('https://decentraland.zone/auth')
+      }
+
+      const connectedAccount = AuthServerProvider.getAccount()
+
+      if (connectedAccount) {
+        connectedAccountRef.current = connectedAccount
+
+        const env = getWantedChainId() === ChainId.ETHEREUM_SEPOLIA ? 'zone' : 'org'
```
Comment (juanmahidalgo) [inline src/components/auth/LoginWithAuthServerPage.tsx:52]:
```suggestion
        const env = getWantedChainId() === ChainId.ETHEREUM_MAINNET ? 'org' : 'zone'
```
Can we do this so if we ever have to migrate from Sepolia to a new one we don't have to update this?

File: src/components/auth/LoginWithAuthServerPage.tsx [hunk 2]
```diff
@@ -75,63 +103,122 @@ export const LoginWithAuthServerPage = () => {
...
+            </div>
+          </>
+        }
+        right={
+          <WearablePreview
+            lockBeta={true}
+            panning={false}
+            disableBackground={true}
+            profile={connectedAccountRef.current}
+            dev={getWantedChainId() === ChainId.ETHEREUM_SEPOLIA}
```
Comment (juanmahidalgo) [inline src/components/auth/LoginWithAuthServerPage.tsx:169]: samehere
```suggestion
            dev={getWantedChainId() !== ChainId.ETHEREUM_MAINNET}
```

File: src/components/auth/LoginWithAuthServerPage.tsx [hunk 3]
```diff
@@ -75,63 +103,122 @@ export const LoginWithAuthServerPage = () => {
...
+                Use a different account
+              </Button>
+            </div>
+          </>
+        }
+        right={
+          <WearablePreview
+            lockBeta={true}
+            panning={false}
+            disableBackground={true}
```
Comment (juanmahidalgo) [inline src/components/auth/LoginWithAuthServerPage.tsx:167]: you could use the short version
```suggestion
            disableBackground
```

**Review by juanmahidalgo — APPROVED** [review]
LGTM! A few small suggestions but looks great

File: src/components/auth/LoginWithAuthServerPage.tsx (hunk 3 above)
Comment (fzavalia) [reply src/components/auth/LoginWithAuthServerPage.tsx]: Did it as well for lockBeta

**Pull request merged**